src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs

```diff
@@ -37,7 +37,7 @@ public static class RecommendedConfig
                     .DontEnforcePowerPlan(); // make sure BDN does not try to enforce High Performance power plan on Windows
 
                 // See https://github.com/dotnet/roslyn/issues/42393
-                job = job.WithArguments(new Argument[] { new MsBuildArgument("/p:DebugType=portable") });
+                job = job.WithArguments(new Argument[] { new MsBuildArgument("/p:DebugType=portable"), new MsBuildArgument("-bl:benchmarkdotnet.binlog") });
```

Comment:
This is for creating binlog file for building bdn generated template project. I feel that it's may be helpful in general for bdn diagnostic purpose.

Reply:
Does it get automatically uploaded with the helix artifacts?

Reply:
I think that the binlog for building the entrypoint executable (in this case is benchmarkdotnet ) is automatically uploaded, but the biglog for building bdn generated project file is not created.

Reply:
Do we move this around to a location where it can be uploaded?